package.json

```diff
@@ -29,11 +29,11 @@
     "install": "node-pre-gyp install --fallback-to-build"
   },
   "dependencies": {
-    "nan": "2.6.2",
-    "node-pre-gyp": "0.6.36"
+    "nan": "^2.7.0",
```

Comment:
I _strongly_ recommend you avoid `^` for dependencies. Patch versions have and will break installs.